nullifiersCounter/nullifiersCounter.circom

```diff
@@ -37,13 +52,20 @@ template CountNullifiers(nullifiersCount, treeDepth) {
     signal input proofsBranches[nullifiersCount][treeDepth];
     // Hashing orders to build elements (contains 0 and 1)
     signal input proofsOrder[nullifiersCount][treeDepth];
+    // Some random number for the whole voting period that is the Poseidon hash of some value. 
+    // This signal is hashed with nullifier and work as commitment for each counter
+    signal input votingHashValue;
 
     // Array with flags that shows if proof is built or not, count of 1 in this output indicates the total amount 
     // of nullifiers that were created for the document
     signal output totalDuplicates;
+    // Commitments for each nullifier
+    signal output commitments[nullifiersCount];
```

Comment:
You are proving that all duplicates come from the same passport preimage, so the commitments will be the same for all proved nullifiers duplication. There is no sense to  add so many additional public signals. Instead add constraints that for all proved nullifiers duplication commitment is constant